scripts/dotnet.py

```diff
@@ -92,6 +92,8 @@ def get_target_framework_moniker(framework: str) -> str:
             return 'net8.0'
         if framework == 'nativeaot9.0':
             return 'net9.0'
+        if framework == 'nativeaot10.0':
+            return 'net10.0'
```

Comment:
This case? I think we can hijack nativeaot9.0 for now to return net10.0 as well?

Reply:
Let's try to add channel to [performance/azure-pipelines.yml](https://github.com/dotnet/performance/blob/f6d0cff8bfb8bc0c3eec23b4b07c837e4c5f8abb/azure-pipelines.yml#L149) - probably we should keep net9 if net8 is still there..

Reply:
Reverted, and update the pipeline.